src/lib/ptree/json.lua

```diff
@@ -57,7 +79,7 @@ end
 local escaped_string_chars =
    { r="\r", n="\n", t="\t", ["\\"]="\\", ['"']='"', b="\b", f="\f", ["/"]="/" }
 
-local function read_json_string(input)
+read_json_string = function(input)
```

Comment:
As stated before, it should be declared as `function fn (args)` instead of `fn = function (args)`, following the style of the rest of the code base.